communication/youtube/youtube-guidelines.md

```diff
@@ -1,17 +1,21 @@
+<!-- omit in toc -->
```

Comment:
Just an FYI -- this will prevent the heading from auto-being added to the ToC by most markdown linters